internal/build/iana_rdap.go

```diff
@@ -0,0 +1,49 @@
+package build
+
+import (
+	"encoding/json"
+)
+
+const (
+	ianaRdapURL = "https://data.iana.org/rdap/dns.json"
+)
+
+// FetchRDAPFromIANA retrieves the map of zones to RDAP service endpoints from IANA.
+func FetchRDAPFromIANA(zones map[string]*Zone) error {
+	res, err := Fetch(ianaRdapURL)
+	if err != nil {
+		return err
+	}
+	defer res.Body.Close()
+
+	x := struct {
+		Services [][][]string // I'm screaming at this schema
```

Comment:
LOL, and also could you put in a comment a minimal example of the schema? What are the levels, just unnamed nested arrays?

Reply:
Yes the schema is unnamed nested arrays, I'll include a minimal example.